src/Workspaces/CSharp/Portable/Classification/ClassificationHelpers.cs

```diff
@@ -289,6 +289,10 @@ private static bool IsVerbatimStringToken(SyntaxToken token)
             {
                 return ClassificationTypeNames.LabelName;
             }
+            else if (token.Parent.IsKind(SyntaxKind.FunctionPointerUnmanagedCallingConvention))
+            {
+                return ClassificationTypeNames.ClassName;
```

Comment:
don't we want to do this if GetCallingConventionSymbol returns non-null?

Reply:
e.g. if you write `delegate[Gibberish]` we wouldn't classify that guy :)

Reply:
@CyrusNajmabadi This matches the compiler classification :)

https://github.com/dotnet/roslyn/blob/eb95c2f45074343b6fa06e723e8dc4cba59d1d6a/src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Members.cs#L588

Quick info relies on compiler classification too:

![image](https://user-images.githubusercontent.com/31348972/152633017-1c8c6415-427c-44eb-ae48-ee03dee1a7d8.png)